dtfs-central-api/src/v1/controllers/portal/facility/patch-submit-amendment.controller.ts

```diff
@@ -38,6 +41,13 @@ export const patchSubmitAmendment = async (req: PatchSubmitAmendmentToUkefReques
           auditDetails,
         });
 
+        // sends email to maker with the details of the amendment
+        await externalApi.sendEmail(EMAIL_TEMPLATE_IDS.PORTAL_AMENDMENT_SUBMITTED_TO_UKEF_MAKER_EMAIL, makersEmail, emailVariables);
```

Comment:
Can this be as follows please.

```suggestion
const emails = [makersEmail, checkerEmail, pimEmail];

emails.forEach((email) =>     externalApi.sendEmail(EMAIL_TEMPLATE_IDS.PORTAL_AMENDMENT_SUBMITTED_TO_UKEF_MAKER_EMAIL, email, emailVariables));
```